### ISSUE TITLE
Can't disconnect from chat servers, client.quit() not working

### ISSUE THREAD

**Issue by Satont** [d-fischer/twitch#128]
## Bug Report
When i calls `await chatClient.quit()` seems like my event loop just stucks 

### Code
That `console.log` will never fires
```ts
await chatClient.quit()
console.log('Disconnected from server')
```
if used not in asynchronous function, then loop not stucked, but also client not disconnected.
```ts
chatClient.quit()
console.log('disconnected from server')
```
This console.log will works.
### Expected behavior
I'm except successful disconnect from chat servers.

### Actual Behavior
loop stucked if used async function and chat client not disconnected from twitch servers

### Environment

- Version: twitch-chat-client: 4.0.10, twitch: 4.0.10
- Node version: 13.12.0
- Operating system: Linux

**Comment by d-fischer** [comment]
Fixed in the `next` branch in 3d01210f, and published as version 4.1.0-pre.9.

**Comment by roncli** [comment]
I can confirm that this is NOT fully fixed in 4.1.0-pre.11.  The promise never resolves, and now results in an unfriendly warning being shown.

`⚠️  ircv3 WARNING Reconnecting because the last ping took over 10 seconds`
